sebs/utils.py

```diff
@@ -168,21 +171,26 @@ class LoggingBase:
     def __init__(self):
         uuid_name = str(uuid.uuid4())[0:4]
         if hasattr(self, "typename"):
-            self.logging = logging.getLogger(f"{self.typename()}-{uuid_name}")
+            self._logging = logging.getLogger(f"{self.typename()}-{uuid_name}")
```

Comment:
We should also use the typename-uuid combination in the colored printer. It makes it much easier to debug experiments, especially the parallel ones.

Old
```
01:11:22,963 INFO AWS.HTTPTrigger-6c46: Invoke of function was successful
```

New, colored
```
[01:11:22.963650] Invoke of function was successful
```

The `INFO` part we do not need :-)

Reply:
Okay, I got it fixed. Now it would output something like this.

![image](https://user-images.githubusercontent.com/16151555/227733967-9063ba65-7a6a-46aa-9ef3-0081cc97fb64.png)